Classes/Controller/ToolboxController.php

```diff
@@ -362,18 +395,55 @@ private function renderImageManipulationTool(): void
      */
     private function renderModelDownloadTool(): void
     {
-        if (
-            $this->isDocMissingOrEmpty()
-            || empty($this->settings['fileGrpsModelDownload'])
-        ) {
-            // Quit without doing anything if required variables are not set.
+        $modelUrl = $this->getModelUrl();
+        if ($modelUrl) {
```

Comment:
Shouldn't this check for NO `$modelUrl` instead?

```suggestion
        if ($modelUrl === '') {
```

Reply:
Great thx!